File: example1/Exercise1.py
import time

def printLines(fileName):
		lines = ''
		with open(fileName) as f:
			lines = f.readlines()

		print lines

if __name__ == "__main__":
	
	printLines('file1.txt')
	printLines('file2.txt')

	print sum(range(10))
	
	



	

File: example1/Exercise1.scala
object Exercise1 {
   import scala.io.Source

   def printLines(filename: String){
	for (line <- Source.fromFile(filename).getLines) {
	   		println(line)
	   }
   }

   def main(args: Array[String]) {
        printLines("file1.txt")
        printLines("file2.txt")

        println(Array.range(0, 10) reduceLeft { _ + _ })
    }

}